src/libs/actions/Report.js

```diff
@@ -528,6 +528,11 @@ function openReport(reportID, participantList = [], newReportObject = {}) {
         }],
     };
 
+    const apiParams = {
+        reportID,
+        emailList: participantList ? participantList.join(',') : '',
+    };
```

Comment:
NAB: for consistency call this params or parameters - we don't use apiParams anywhere else